__tests__/api.test.js

```diff
@@ -0,0 +1,38 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("/api", () => {
+  it("GET - 200: Responds with message 'connection ok'", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toEqual({
+          message: "connection ok",
+        });
+      });
+  });
+});
```

Comment:
One of the tickets is actually to make a `/api` endpoint, but its purpose is a little different from this!

I would recommend you keep the logic of this endpoint, since I think it serves a valid purpose, but put it on `api/health-check` or something similar to that.

Reply:
Done